TrainingApp/Model/Flight.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Flight.swift
+//  TrainingApp
+//
+//  Created by Dagmawit Alemayehu on 28/02/2023.
+//
+
+struct Flight: Codable {
+    
+}
```

Comment:
are you sure that this structure is necessary? because it's empty

Reply:
It's a new struct I will use for parsing a JSON API result